README.md

```diff
@@ -148,6 +148,12 @@ Add css rules in your stylesheet
 }
 ```
 
+<h4>Using the <code>img</code> tag and referencing a svg directly from the repo.</h4>
+
+```html
+<img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/aftereffects/aftereffects-original.svg'>
```

Comment:
This is not recommended. Instead use a CDN like jsdelivr. And I would suggest to link the devicon icon instead of aftereffects.

Reply:
That sounds like a good idea. I just grabbed the thing haha